afteraction-share.html

```diff
@@ -396,8 +396,11 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 						data: $('#action-form').serialize()
 					})
 					.done(function(msg) {
-						// send google analytics event
-						ga('send', 'event', 'scrolling-signup', 'signup', window.location.pathname);
+						// handle adblockers preventing the loading of Google Analytics library
```

Comment:
maybe would normally want to make this a separate function call that has the check in it. BUT i actually think what you have in here is fine for this case
might wanna also add the ga object check around the ga send at: 
https://github.com/350org/ak_intl_v3/blob/bf111321d2dae3f09fb4ba14372eb29c92434a31/afteraction-share.html#L372
even though it is outside this ticket's explicit scope?